Description: Add libargyll.la as a linked library where it is needed. Closes #615692.
Author: Mònica Ramírez Arceda <monica@probeta.net>
Last-Update: 2011-06-10

Index: argyll-1.1.1/target/Makefile.am
===================================================================
--- argyll-1.1.1.orig/target/Makefile.am	2011-06-10 18:35:24.211998229 +0200
+++ argyll-1.1.1/target/Makefile.am	2011-06-10 18:36:09.895998189 +0200
@@ -11,7 +11,7 @@
 	$(ICC_LIBS) ../render/librender.la ../cgats/libcgats.la	\
 	../xicc/libxicc.la ../gamut/libgamut.la			\
 	../spectro/libinsttypes.la ../spectro/libconv.la	\
-	../numlib/libargyllnum.la $(TIFF_LIBS)
+	../numlib/libargyllnum.la ../libargyll.la $(TIFF_LIBS)
 
 bin_PROGRAMS = targen printtarg
 
Index: argyll-1.1.1/link/Makefile.am
===================================================================
--- argyll-1.1.1.orig/link/Makefile.am	2011-06-10 18:35:24.223998158 +0200
+++ argyll-1.1.1/link/Makefile.am	2011-06-10 18:36:09.895998189 +0200
@@ -6,7 +6,7 @@
 	../xicc/libxutils.la ../gamut/libgamut.la			\
 	../gamut/libgammap.la ../plot/libplot.la ../plot/libvrml.la	\
 	../rspl/librspl.la $(ICC_LIBS) ../cgats/libcgats.la		\
-	../numlib/libargyllnum.la $(X_LIBS) $(TIFF_LIBS)
+	../numlib/libargyllnum.la ../libargyll.la $(X_LIBS) $(TIFF_LIBS)
 
 bin_PROGRAMS = collink pathplot
 
